docs/en_us/ingress/redirect.md

```diff
@@ -0,0 +1,103 @@
+# Redirect
+
+The BFE Ingress Controller supports redirecting traffic matched by the Ingress by using `metadata.annotations` of the Ingress object.
+
+## How to Config
+
+In the Ingress object,
+
+- `spec.rules` defines the route rules;
+- `metadata.annotations` defines the behavior of redirecting traffic matched by the Ingress.
+
+Reference format:
+
+```yaml
+metadata:
+  annotations:
+    bfe.ingress.kubernetes.io/redirect.url-set: "https://www.baidu.com"
+spec:
+  rules:
+  - ...
+```
+
+```yaml
+metadata:
+  annotations:
+    bfe.ingress.kubernetes.io/redirect.scheme-set: https
+    bfe.ingress.kubernetes.io/redirect.status: 301
+spec:
+  rules:
+  - ...
+```
+
+## Redirect Location
+
+The BFE Ingress Controller supports 4 ways to configure the redirect location.
+
+### 静态URL
+
+Use `bfe.ingress.kubernetes.io/redirect.url-set` to config the static redirect location。
+
+For example:
+
+```yaml
+bfe.ingress.kubernetes.io/redirect.target: "https://www.baidu.com"
+```
+
+Corresponding scenario:
+
+- Request: http://host/path?query-key=value
+- Response: https://www.baidu.com
+
+### Fetch URL from Query
+
```

Comment:
Redirect location is fetched from specific `Query` of request URL, query key is specified by `bfe.ingress.kubernetes.io/redirect.url-from-query`.